libs/application/templates/complaints-to-althingi-ombudsman/src/fields/ComplaintOverview/ComplaintOverview.tsx

```diff
@@ -71,11 +75,12 @@ export const ComplaintOverview: FC<FieldBaseProps> = ({ application }) => {
         />
       </ReviewGroup>
       <ReviewGroup isLast>
-        <Text variant="h5">Fylgiskjöl</Text>
+        <Text variant="h5"></Text>
```

Comment:
Was this accidentally leftover?

Reply:
Yes good catch, was supposed to be deleted. Fixed